core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -482,11 +533,26 @@ public static String requestContextStorageProvider() {
      *
      * <p>This flag is enabled by default for supported platforms. Specify the
      * {@code -Dcom.linecorp.armeria.useEpoll=false} JVM option to disable it.
+     *
+     * @deprecated Use {@link #defaultTransportType()} and {@code -Dcom.linecorp.armeria.transportType=epoll}.
      */
+    @Deprecated
     public static boolean useEpoll() {
         return USE_EPOLL;
     }
 
+    /**
+     * Returns the {@link TransportType} that will be used for socket I/O in Armeria.
+     *
+     * <p>The default value of this flag is {@link #DEFAULT_TRANSPORT_TYPE_SPEC}, which retains
```

Comment:
```
... this flag is {@code "epoll"} in Linux and {@code "nio"} for other operations systems.
Specify ...
```

('... which retains' doesn't seem to add anything new, so I think we can remove it.)